game.tscn

```diff
@@ -132,15 +115,15 @@ stream = ExtResource("5_nrkj6")
 [node name="ReboundAudioPlayer" type="AudioStreamPlayer" parent="."]
 stream = ExtResource("6_gibg6")
 
-[node name="Menu" parent="." instance=ExtResource("7_vv8hj")]
+[node name="CanvasLayer" type="CanvasLayer" parent="."]
```

Comment:
Do you know why UI anchoring seems to only work great when the control type nodes are children of a CanvasLayer node?  In Game #2 I was making a top score horizontal panel and I couldn't get it to properly show until I've put it into a CanvasLayer.